MeetupPlatformApi/Persistence/Migrations/20220407135015_AddUsersByRoles.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace MeetupPlatformApi.Migrations
+{
+    public partial class AddUsersByRoles : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "Discriminator",
+                table: "users",
+                type: "text",
+                nullable: false,
+                defaultValue: "");
+
+            migrationBuilder.AddColumn<Guid>(
+                name: "meetup_id",
+                table: "users",
+                type: "uuid",
+                nullable: true);
+
+            migrationBuilder.AddColumn<Guid>(
+                name: "organizer_id",
+                table: "meetups",
+                type: "uuid",
+                nullable: false);
+
+            migrationBuilder.CreateIndex(
+                name: "IX_users_meetup_id",
+                table: "users",
+                column: "meetup_id");
+
+            migrationBuilder.CreateIndex(
+                name: "IX_meetups_organizer_id",
```

Comment:
- All other indices are snake case, starting with the ix prefix followed by the containing table name and all included fields: ix_meetups_organizer_id